### DISCUSSION TITLE
v3.15.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#10448] [Releases & Announcements]
## Changelog

- ip-lookup database lazy loading for all installation environments
- New sidebar for Chatwoot ( v4 )
- New design for HelpCenter portals ( v4 )
- New design for campaigns page ( v4 )
- Numerous bug fixes and enhancements in preparation for the v4 release

Thanks to @mazenkhalil, @KokeroO, @MingShyanWei for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.15.0'>v3.15.0</a>.</em>

**Comment by johnnyvernin** [comment]
Dear friends, I'm running on a docker, after update from 3.12.0 to 3.15.0), can't login anymore (all admin users got error):

`I, [2024-12-05T00:13:44.299451 #1]  INFO -- : [b835e6e8-15e5-4f08-b5ba-88a52060101c] Started POST "/auth/sign_in" for 138.59.153.59 at 2024-12-05 00:13:44 +0000

I, [2024-12-05T00:13:44.322632 #1]  INFO -- : [b835e6e8-15e5-4f08-b5ba-88a52060101c] Processing by DeviseOverrides::SessionsController#create as HTML

I, [2024-12-05T00:13:44.327219 #1]  INFO -- : [b835e6e8-15e5-4f08-b5ba-88a52060101c]   Parameters: {"email"=>"myadminemail@gmail.com", "password"=>"[FILTERED]", "sso_auth_token"=>"[FILTERED]"}

I, [2024-12-05T00:13:45.518260 #1]  INFO -- : [b835e6e8-15e5-4f08-b5ba-88a52060101c] Completed 500 Internal Server Error in 1191ms (ActiveRecord: 223.8ms | Allocations: 63254)

F, [2024-12-05T00:13:45.521684 #1] FATAL -- : [b835e6e8-15e5-4f08-b5ba-88a52060101c]   

[b835e6e8-15e5-4f08-b5ba-88a52060101c] ActionView::Template::Error (undefined method `custom_role_id' for an instance of AccountUser):

[b835e6e8-15e5-4f08-b5ba-88a52060101c]     1: json.custom_role_id account_user&.custom_role_id

[b835e6e8-15e5-4f08-b5ba-88a52060101c]     2: json.custom_role account_user&.custom_role&.as_json(only: [:id, :name, :description, :permissions])

[b835e6e8-15e5-4f08-b5ba-88a52060101c]   

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/views/api/v1/models/_user.json.jbuilder:33

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/views/api/v1/models/_user.json.jbuilder:21

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/views/api/v1/models/_user.json.jbuilder:20

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/views/devise/_auth.json.jbuilder:2

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/views/devise/_auth.json.jbuilder:1

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/controllers/devise_overrides/sessions_controller.rb:23:in `render_create_success'

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/controllers/devise_overrides/sessions_controller.rb:18:in `create'

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/controllers/concerns/switch_locale.rb:39:in `set_locale'

[b835e6e8-15e5-4f08-b5ba-88a52060101c] app/controllers/concerns/switch_locale.rb:12:in `switch_locale'`

Could anyone help me?

Thanks!

**Reply by sojan-official** [reply]
@johnnyvernin  seems like you haven't run the db migrations, please make sure to do that and restart the nodes

**Comment by hexclann** [comment]
I have a selfhosted version of. chatwoot v3.15.0 installed on a standalone ubuntu VM. I have 4 users and 5 accounts. But when I try to create any additional users from the super_admin page I get an error "User limit reached. Please purchase more licenses from super admin". Do I need to purchase a license? Is there any configuration Im missing?

**Reply by sojan-official** [reply]
please reach out using the contact support option in your super admin and via the live chat on website so that the team can take a look into this.
